activestate.yaml

```diff
@@ -433,3 +433,4 @@ events:
   - name: file-changed
     scope: ["internal/locale/locales"]
     value: build
+config_version: 1
```

Comment:
Someones opted in :p We shouldn't commit this just yet. The intend is that you'd commit this along with the buildscript, which isn't quite ready yet.